modules/installation-guide/partials/proc_configuring-workspace-exposure-strategies-using-a-helm-chart.adoc

```diff
@@ -25,3 +28,28 @@ $ helm upgrade --set global.serverStrategy=__<single-host>__
 ----
 
 Depending on the strategy used, replace the `_<single-host>_` option in the above example with `multi-host` or `default-host`.
+
+
+Example for `gateway` single-host strategy type:
+
+[subs="+quotes"]
+----
+$ helm install --namespace __<namespace>__ --set global.serverStrategy=single-host --set global.singleHostExposure=gateway --set cheSinglehostGateway.deploy=true
+----
+
+or:
+
+[subs="+quotes"]
+----
+$ helm upgrade --install che --namespace __<namespace>__ --set global.serverStrategy=single-host --set global.singleHostExposure=gateway --set cheSinglehostGateway.deploy=true
+----
+
+Alternatively, predefined values file can be used as follows:
+
+[subs="+quotes"]
+----
+$ helm upgrade --install che --namespace __<namespace>__ \
+--set global.ingressDomain=$(minikube ip).nip.io \
```

Comment:
ingressDomain is not single-host specific thing and you don't have it in previou examples.